app/index.html

```diff
@@ -55,8 +55,8 @@
 
     <!-- build:js(.) scripts/vendor.js -->
     <!-- bower:js -->
-    <script src="bower_components/jquery/dist/jquery.js"></script>
     <script src="bower_components/angular/angular.js"></script>
+    <script src="bower_components/jquery/dist/jquery.js"></script>
```

Comment:
Was having jQuery first causing something to fail?  This way means AngularJS will be using its own jqLite for `$element`, right?  Which is fine, except in those occasional instances where one wants to do something unusual with `$element` and having the full jQuery comes in handy.

Reply:
Hmm. Good eye. I didn't manually change this, the order was modified by the package updates I did. I'll see about reverting it.